Library/Formula/elixirscript.rb

```diff
@@ -0,0 +1,21 @@
+class Elixirscript < Formula
+  desc "Elixir to JavaScript compiler"
+  homepage "https://github.com/bryanjos/elixirscript"
+  url "https://github.com/bryanjos/elixirscript/archive/v0.16.0.tar.gz"
+  sha256 "1ccc52501be1d762e823c6174aa5db1de200f4edc96b39f1057fa8aefeb6212e"
+
+  depends_on "elixir" => :build
+
+  def install
+    system "mix", "local.hex", "--force"
+    system "mix", "deps.get"
+    system "mix", "escript.build"
+    bin.install "elixirscript"
+    prefix.install "priv/Elixir.js", "LICENSE"
+  end
+
+  test do
+    assert_match /Symbol.for('atom')/,
+                 shell_output("#{bin}/elixirscript \":atom\" -ex")
```

Comment:
Just indent this by two characters. Also, this test is failing: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/43223/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/test_elixirscript/